frontend/appflowy_flutter/lib/workspace/presentation/settings/pages/settings_manage_data_view.dart

```diff
@@ -0,0 +1,491 @@
+                ),
+              ],
+              SettingsCategory(
+                title: LocaleKeys.settings_manageDataPage_cache_title.tr(),
+                children: [
+                  SingleSettingAction(
+                    labelMaxLines: 4,
+                    label: LocaleKeys.settings_manageDataPage_cache_description
+                        .tr(),
+                    buttonLabel:
+                        LocaleKeys.settings_manageDataPage_cache_title.tr(),
+                    onPressed: () {
+                      SettingsAlertDialog(
+                        title: LocaleKeys
+                            .settings_manageDataPage_cache_dialog_title
+                            .tr(),
+                        subtitle: LocaleKeys
+                            .settings_manageDataPage_cache_dialog_description
+                            .tr(),
+                        confirm: () async {
+                          await getIt<FlowyCacheManager>().clearAllCache();
+                          if (context.mounted) {
+                            showSnackBarMessage(
+                              context,
+                              LocaleKeys
+                                  .settings_manageDataPage_cache_dialog_successHint
+                                  .tr(),
+                            );
+                            Navigator.of(context).pop();
+                          }
+                        },
+                      ).show(context);
+                    },
+                  ),
+                ],
+              ),
+              // Uncomment if we need to enable encryption
+              //   if (userProfile.authenticator == AuthenticatorPB.Supabase) ...[
+              //     const SettingsCategorySpacer(),
+              //     BlocProvider(
+              //       create: (_) => EncryptSecretBloc(user: userProfile),
+              //       child: SettingsCategory(
+              //         title: LocaleKeys.settings_manageDataPage_encryption_title
+              //             .tr(),
+              //         tooltip: LocaleKeys
+              //             .settings_manageDataPage_encryption_tooltip
+              //             .tr(),
+              //         description: userProfile.encryptionType ==
+              //                 EncryptionTypePB.NoEncryption
+              //             ? LocaleKeys
+              //                 .settings_manageDataPage_encryption_descriptionNoEncryption
+              //                 .tr()
+              //             : LocaleKeys
+              //                 .settings_manageDataPage_encryption_descriptionEncrypted
+              //                 .tr(),
+              //         children: [_EncryptDataSetting(userProfile: userProfile)],
+              //       ),
+              //     ),
+              //   ],
+            ],
+          );
+        },
+      ),
+    );
+  }
+}
+
+// class _EncryptDataSetting extends StatelessWidget {
```

Comment:
I've left a lot of commented code in the settings for now, since there's a few settings from the design that I think we don't need yet, or we don't have support for yet.